include/SFML/Window/Event.hpp

```diff
@@ -235,30 +339,33 @@ struct Event
 ///
 /// A sf::Event instance contains the type of the event
 /// (mouse moved, key pressed, window closed, ...) as well
-/// as the details about this particular event. Please note that
-/// the event parameters are defined in a union, which means that
-/// only the member matching the type of the event will be properly
-/// filled; all other members will have undefined values and must not
-/// be read if the type of the event doesn't match. For example,
-/// if you received a KeyPressed event, then you must read the
-/// event.key member, all other members such as event.mouseMove
-/// or event.text will have undefined values.
+/// as the details about this particular event. Each event
+/// corresponds to a different struct which contains the data
+/// required to process that event.
+///
+/// The way to access the current active event is via
+/// sf::Event::getIf which returns the address of the event struct
+/// if it's active or a nullptr otherwise. sf::Event::is is used
+/// to check the active event type without actually reading any
+/// of the corresponding event data. These are useful if you need
+/// to check only a single possible active event or prefer to
+/// process events in a series of if/else if blocks.
```

Comment:
Does not apply anymore as there are no other methods.
```suggestion
/// of the corresponding event data.
```